Tests/Unit/Domain/Repository/TeaRepositoryTest.php

```diff
@@ -25,7 +25,7 @@ protected function setUp(): void
             // @phpstan-ignore-next-line This line is 11LTS-specific, but we're running PHPStan on TYPO3 12.
             $this->subject = new TeaRepository($objectManagerStub);
         } else {
-            $this->subject = new TeaRepository();
+            $this->subject = new \TTN\Tea\Domain\Repository\TeaRepository();
```

Comment:
Why do we introduce the FCQN for this line while using the imported namespace on line 26?

Reply:
Done.